hack/validate-imports/validate-imports.go

```diff
@@ -22,16 +29,69 @@ func validateUnderscoreImport(path string) error {
 
 	switch path {
 	case "net/http/pprof",
-		"github.com/Azure/ARO-RP/pkg/util/scheme":
+		"github.com/Azure/ARO-RP/pkg/util/scheme",
+		"embed":
 		return nil
 	}
 
 	return fmt.Errorf("invalid _ import %s", path)
 }
 
-// acceptableNames returns a list of acceptable names for an import; empty
+type importValidator struct {
+	AllowedNames map[string][]string `json:"allowedImportNames"`
+}
+
+func initValidator() importValidator {
+	allowed := importValidator{}
+	err := yaml.Unmarshal(allowedNamesYaml, &allowed)
+	if err != nil {
+		log.Println(err)
+		return importValidator{}
```

Comment:
why do we return the "empty", but initialized validator instead off err-ing out here? could this give us some false positive CI passing?

Reply:
I agree. It now exits with status code 1 :)